geohosting/api.py

```diff
@@ -48,6 +48,23 @@ def get_csrf_token():
     return {
         "csrf_token": frappe.sessions.get_csrf_token()
     }
-    
 
+@frappe.whitelist(allow_guest=False)
```

Comment:
@ketankartoza can you move this method to support tickets.py and call it from there

Reply:
ok